pages/rsvp.tsx

```diff
@@ -17,33 +20,38 @@ const initialSearchForm = {
   },
 };
 
-const Container = tw.div`relative w-full h-full mt-28`;
+const Container = tw.div`relative w-full h-full`;
 
 const Rsvp = () => {
   const dispatch = useDispatch();
 
+  const { maintenanceEnabled } = useSelector((state) => state.maintenanceMode);
   const guests = useSelector((state) => state.guests.guests);
   const parties = useSelector((state) => state.guests.parties);
   const selectedParty = useSelector((state) => state.guests.selectedParty);
 
-  const [searchForm, updateSearchForm] = useState<GuestSearchForm>(initialSearchForm);
+  const [searchForm, updateSearchForm] =
+    useState<GuestSearchForm>(initialSearchForm);
   const [searchResults, setSearchResults] = useState<Models.Guest[]>([]);
 
   // copied from SaveTheDate.tsx, should move to util file?
-  const selectParty = useCallback((id: string) => {
-    const party = parties.find((p) => p.id === id);
-    if (party) {
-      dispatch(setSelectedParty(party));
-    }
-  }, [parties, dispatch]);
+  const selectParty = useCallback(
```

Comment:
[non-blocking]: Wondering if there's a way to define this in a util file of some sort to DRY up code since the SaveTheDate uses this? 🤔 . Not urgent, and prob not even necessary, but the clean code ocd in me kicking in haha. I can try and DRY up code in another PR too

Reply:
Yeah, definitely.